inspector/ios-inspector/ForgeModule/tabs_WKWebViewDelegate.m

```diff
@@ -51,6 +51,25 @@ - (void) webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigat
     return decisionHandler(WKNavigationActionPolicyAllow);
 }
 
+- (void)webView:(WKWebView *)webView decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler {
+    
+    NSURLResponse* response = [navigationResponse response];
+    if ([response isKindOfClass:[NSHTTPURLResponse class]]) {
+       NSHTTPURLResponse* httpResponse = (NSHTTPURLResponse*) [navigationResponse response];
+       //create cookie array from http response
+        NSDictionary* allHeaderFields = [httpResponse allHeaderFields];
```

Comment:
The `allHeaderFields` doesn't contain any `set-cookie` headers, therefore the subsequent code has no affect.

You can try it here: https://req-playground.scthi.tech/cookies
Every response sets the current timestamp to cookie with name `LastVisit`.

![Screenshot 2020-01-03 at 08 29 57](https://user-images.githubusercontent.com/318533/71711884-4378ef00-2e03-11ea-87ed-df2ba6d48896.png)